test/functional/ui/screen.lua

```diff
@@ -205,6 +205,8 @@ end
 
 function Screen:try_resize(columns, rows)
   uimeths.try_resize(columns, rows)
+  -- Give ourselves a chance to _handle_resize
+  self:sleep(0.1)
```

Comment:
There a decent number of tests that perform resizes.  Without this, the `screen:expect` calls after `try_resize` would trip the assert I added because `_handle_resize` hadn't been called and therefore hadn't updated `_height` yet.